pkg/kudoctl/util/repo/package.go

```diff
@@ -20,9 +20,9 @@ import (
 )
 
 const (
-	frameworkV1FileName     = "framework.yaml"
-	templateV1FileNameRegex = "templates/.*.yaml"
-	paramsV1FileName        = "params.yaml"
+	v1FrameworkFileName     = "framework.yaml"
```

Comment:
I think we can drop even the v1. I know it's not related to your PR but since you already touched it... It's leftover from my PR...

Reply:
So you're saying, we'll never have `v2`? :) But if you say so...